example/transition_vtol_fixed_wing/transition_vtol_fixed_wing.cpp

```diff
@@ -1,72 +1,50 @@
-
 #include <chrono>
 #include <cstdint>
+#include <iostream>
```

Comment:
Just to mention it: my preference for headers is to include them in alphabetical order. `dronecode` is a dependency of the example, just as `thread`. Alphabetical order is always consistent, where the preference of the developer, well, depends on the developer :-).

Reply:
Right, but I'm the correct developer :smile: